doc/f5-oslbaasv1-readme_edits.md

```diff
@@ -0,0 +1,889 @@
+---
+layout: default
+title: F5 LbaaS Plugin for Neutron LBaaS v2.0.1
+---
```

Comment:
This is not the correct file for the 1.0 branch. It should be for 1.0.12, which will work on Havana-Kilo. Looks like you may have copied in the file from liberty branch instead of applying a commit (diff) incrementally to the right file in the 1.0 branch.